src/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum EVENT_STATUS_OPTIONS {
+  Draft = 'Draft',
+  Published = 'Published',
+}
```

Comment:
Hmm, never seen this naming convention before. Typically enums are named as such:
```ts
export enum EventStatus{
  DRAFT = 'DRAFT',
  PUBLISHED = 'PUBLISHED',
}
```

Reply:
I dont have a lot of experience with enums, I tried to follow the conventions from https://www.typescriptlang.org/docs/handbook/enums.html. Also the enum is stored in the back-end as "Draft" and "Published". How do you recommend changing the names in this situation? Should all of the fields be capitalized in both front-end and back-end?

Reply:
Just checked out the handbook and enum are also named using PascalCase there too?

Regarding the point about how the values are stored in the backend, you can change the values to be "Draft" and "Published", that should be fine. I.e.:
```suggestion
export enum EventStatus {
  DRAFT = 'Draft',
  PUBLISHED = 'Published',
}
```